pkg/ingress/controller/controller.go

```diff
@@ -350,6 +350,13 @@ func (c *Controller) nodeSyncLoop() {
 
 	log.Infof("Detected change in list of current cluster nodes. New node set: %v", nodeNames(newNodes))
 
+	// if no new nodes, then avoid update member
+	if len(newNodes) == 0 {
+		c.knownNodes = newNodes
```

Comment:
if no nodes now, the lb pool needs to be updated, no?

Reply:
let me dig more here... originally I think nodes == [] we should avoid update lb
but another thought here might defer the check to later...